parlai/tasks/saferdialogues/build.py

```diff
@@ -10,13 +10,13 @@
 import parlai.core.build_data as build_data
 import os
 
-SAFERDIALOGUES_VERSION = "v0.1"
+SAFERDIALOGUES_VERSION = "v0.3"
 
 RESOURCES = [
     DownloadableFile(
         f'http://parl.ai/downloads/saferdialogues/saferdialogues_{SAFERDIALOGUES_VERSION}.tar.gz',
         f'saferdialogues_{SAFERDIALOGUES_VERSION}.tar.gz',
-        '7ecf062fdf3e0ddfd16c52998341f543331e0dbaa1fcdb27026e8b0b80446fed',
+        '3d1bc731fb0c63d9f61a52b3d5f1aab0911cda1ca38f49822c79c18a2dc8e834',
```

Comment:
will the new teacher be overriding the previous version (the one with the old valid and test split), I wonder if it would be nicer to keep both splits (one for reproducibility in your paper, and one here)